src/main/scala/com/softwaremill/reactive/complete/LargestDelayActorComplete.scala

```diff
@@ -16,6 +16,18 @@ class LargestDelayActorComplete extends PersistentActor with ActorSubscriber wit
     override def inFlightInternally = inFlight
   }
 
+  override def receive = {
```

Comment:
Did something not work without that method? As we are using a persistent actor, the `receive` method is implemented in the `PersistenActor` trait, and the user should implement `receiveCommand` and `receiveRecover` (as below)